cmd/bosun/database/database.go

```diff
@@ -40,6 +40,9 @@ type DataAccess interface {
 
 	Search_AddMetricTagSet(metric, tagSet string, time int64) error
 	Search_GetMetricTagSets(metric string, tags opentsdb.TagSet) (map[string]int64, error)
+
+	Search_BackupLastInfos(map[string]map[string]*LastInfo) error
```

Comment:
The whole naming is odd. I was trying to segregate the search methods, but there are too many now. I will probably segregate to data.Search.GetWhatever() with a nested interface.